cli/src/commands/deployment/execute.rs

```diff
@@ -118,11 +111,12 @@ struct CdPipelineWithBuilds {
     last_deployed_at: Option<i64>,
     last_successfully_deployed_artifact: Option<String>,
     status: Option<String>,
-    jenkins_builds: Vec<JenkinsBuild>,
+    jenkins_builds: Vec<CdPipelineBuild>,
+    github_builds: Vec<CdPipelineBuild>,
```

Comment:
I'm thinking how about just `builds`? Since we only use either `jenkins_builds` or `github_builds` at one time. What do you think?